content/cpp/concepts/math-functions/terms/rint/rint.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'rint()'
+Description: 'The rint() function in C++ rounds the argument to an integral value using the current rounding mode.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+## Syntax
+
+```pseudo
+rint(x)
+```
+
+Argument must be a `double`/`float`/`long double`, and the return value will be same type.
+
+## Return value
```

Comment:
Remove the "Return Value" header and include the following sentence at the end of the "Syntax" section.

Reply:
@chiragcode11, This comment has not been resolved.